lib/blocs/turbo_payment/turbo_payment_bloc.dart

```diff
@@ -0,0 +1,165 @@
+import 'package:ardrive/blocs/turbo_payment/utils/storage_estimator.dart';
+import 'package:ardrive/services/turbo/payment_service.dart';
+import 'package:ardrive/utils/logger/logger.dart';
+import 'package:arweave/arweave.dart';
+import 'package:equatable/equatable.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import 'file_size_units.dart';
+
+part 'turbo_payment_event.dart';
+part 'turbo_payment_state.dart';
+
+final presetAmounts = [25, 50, 75, 100];
+final supportedCurrencies = ['usd'];
+
+const oneGigbyteInBytes = 1024 * 1024 * 1024;
```

Comment:
```suggestion
const gibibyteInBytes = 1024 * 1024 * 1024;
```
The name is wrong :D